pkg/operators/oci-handler/oci.go

```diff
@@ -148,7 +152,8 @@ func getPullSecret(pullSecretString string, gadgetNamespace string) ([]byte, err
 func (o *ociHandler) InstantiateDataOperator(gadgetCtx operators.GadgetContext, instanceParamValues api.ParamValues) (
 	operators.DataOperatorInstance, error,
 ) {
-	ociParams := apihelpers.ToParamDescs(o.InstanceParams()).ToParams()
+	ociParams := apihelpers.ToParamDescs(o.GlobalParams()).ToParams()
+	*ociParams = append(*ociParams, *apihelpers.ToParamDescs(o.InstanceParams()).ToParams()...)
```

Comment:
```suggestion
         var ociParams *params.Params
	*ociParams = append(*o.globalParams, *apihelpers.ToParamDescs(o.InstanceParams()).ToParams()...)
```

To use the params we initialized using the config.

Reply:
I see perhaps something like this will do the trick:

```
	ociParams := o.globalParams
	if ociParams == nil {
		ociParams = apihelpers.ToParamDescs(o.GlobalParams()).ToParams()
	}
```
I mean if we have globalParams already (and adjusted according to config) we should use them otherwise fallback to creating new ones?

Reply:
Great, thanks!